Detectors/CTP/workflow/src/ReaderSpec.cxx

```diff
@@ -0,0 +1,133 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include "CTPWorkflow/ReaderSpec.h"
```

Comment:
From the filename is it not clear that this is a digit reader, better name it explicitly as DigitReaderSpec.
Also, note that we started to move gradually all readers and writers to separate IO directories, in order to avoid circular dependencies in global workflows.
Perhaps you could place this class directly in the CTP/WorkflowIO .